docker-entrypoint.sh

```diff
@@ -5,4 +5,4 @@ if [ -n "$SECRETS_S3_BUCKET" ]; then
     aws s3 sync s3://$SECRETS_S3_BUCKET/ /secrets
 fi
 
-yarn start_service
```

Comment:
Don't think we want this, not sure nodemon will actually get installed as its a dev dependency?